File: Assets/preload/preload_cards.js
#pragma strict

private var init:Vector3;
private var targ:Vector3;
private var time0:float;
var turn=true;


function Start () {
//	time0=Time.time;
	GetComponent.<Rigidbody>().velocity=Vector3.down*10;
}

function FixedUpdate () {
	if(turn==true) {transform.Rotate(Vector3(9,6.5,0));}//*Time.realtimeSinceStartup);}
}

function OnTriggerEnter (other : Collider) {
	GetComponent.<Rigidbody>().velocity=Vector3.zero; turn=false; Destroy(GetComponent.<Collider>());
}

File: Assets/preload/PreloadCards.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class PreloadCards : MonoBehaviour
{
    private bool isTurning = true;
    private Vector3 turnVector = new Vector3(440, 320, 0);

    void Start()
    {
        GetComponent<Rigidbody>().velocity = Vector3.down * 10;
    }

    void Update()
    {
        if (isTurning)
        {
            transform.Rotate(turnVector * Time.deltaTime);
        }
    }

    void OnTriggerEnter(Collider other) {
    	GetComponent<Rigidbody>().velocity = Vector3.zero;
        isTurning = false;
        Destroy(GetComponent<Collider>());
    }
}
